hydroshare/local_settings.cutover

```diff
@@ -193,3 +193,8 @@ with open(os.path.dirname(os.path.abspath(__file__)) + "/../config/hydroshare-co
     except yaml.YAMLError as exc:
         print(exc)
 
+# HydroShare web services settings
+HSWS_URL = "http://10.37.142.23:8060/his/services/update"
+HSWS_API_TOKEN = "d1caabf3059ef7ffd99d890aba57100da2e0efc8"
+HSWS_TIMEOUT = 3
+HSWS_PUBLISH_URLS = False
```

Comment:
@kjlippold can you please also add to local_settings.template? This is because local_settings.cutover eventually needs to go away and most items will be moved to settings.py, but this is appropriate for local_settings.py, local_settings.cutover, local_settings.template

Wasn't my first choice, but was the optimal (least painful) path forward, given all of the legacy baggage and restrictions on teams across all PIs, organizations, research and companies.

Reply:
@cuahsimarko I still don't quite understand why we need both local_settings.cutover and local_settings.template? Couldn't local_settings.cutover serve also as settings.template? Ultimately, developers just need a local_settings.py to customize to their needs. Why do we need both cutover and template? What is the difference between the two?

Reply:
local_settings.cutover is to deal with the deployment scripts that require a local_settings file. Those deployment scripts need to be updated. local_settings.template contains all of the typical things a developer may want to control, a quickstart.

Reply:
While more familiar developers may have memorized all of the settings, newer developers or contractors will see this as a convenience and assistive in familiarizing with the codebase. I've seen it done on other projects.

Reply:
@cuahsimarko In my local development, I still use local_settings.cutover as a template to copy to local_settings.py, and it works in my local development also. So probably we could consolidate template and cutover to be one file that could be used as a template to further customize to individual environment, local development environment or production environment, as needed. That would reduce confusion in my opinion.

Reply:
@hyi Let's discuss outside of this PR... basically .cutover is just a hodgepodge of legacy and changes incrementally that may not have been thought out. Someone needs to move them out to settings.py and then thoughtfully pull out any remaining items that belong in local_settings.template, which has been deliberately built from first principles.

Reply:
That's the easy part, the hard part is finding and updating everywhere in the shell scripts across build scripts and the repo, itself, then testing everything. This is why this tech debt item (local_settings.py in gitignore, but still tracked and versioned) was never addressed...